contracts/moda-contracts/src/interfaces/IMembership.sol

```diff
@@ -2,13 +2,6 @@
 pragma solidity 0.8.21;
 
 interface IMembership {
-    /**
-     * @dev Updates the merkle root
-     * @param merkleRoot The new merkle root
-     * @notice restricted by access manager
-     */
-    function updateMerkleRoot(bytes32 merkleRoot) external;
-
     /**
      * @dev Checks if an address is whitelisted using a merkle proof
```

Comment:
This comment should not reference "Merkle Proof" or "Whitelisted". Maybe keep it simple:

`Return the membership status of a user for your organization.`